.github/workflows/unit-test.yml

```diff
@@ -70,13 +70,6 @@ jobs:
           go install sigs.k8s.io/kind@v0.19.0
           kind create cluster
 
-      - name: install Kubebuilder
-        uses: RyanSiu1995/kubebuilder-action@7170cb0476187070ae04cbb6cee305e809de2693
-        with:
-          version: 3.9.1
-          kubebuilderOnly: false
-          kubernetesVersion: v1.26.2
-
```

Comment:
does it still work if we remove this kubebuilder action?

Reply:
Yes, I updated `make test` to download the kubebuilder via envtest.

So running on local machine or Github CI should yield the same result.